app/views/initiatives/_initiative.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+json.merge! initiative.as_json
+json.owner do
+  json.id initiative.owner_id
+end
```

Comment:
> If you want to return something super custom you can use Jbuilder

I wouldn't qualify this as super custom as it's only on field tht can be generated by Rails. It's good that you did it to practice but in this context is a little of over engineering